programs/ledger-vault/src/instructions/withdraw.rs

```diff
@@ -21,6 +22,8 @@ pub struct Withdraw<'info> {
     #[account(
```

Comment:
I didn't put any constraints like

```
        associated_token::mint = mint,
        associated_token::authority = vault_state,
```

but it's still working why ?        
same question for user_data

Reply:
> Because without constraints, it is not performing any seeds validation... adding the constraints means that anchor will verify the correctness of the address

Thanks.!I still have something unclear: without the constraints does that mean that someone could call my function with any `user_data` address and steal the tokens for instance ?